WordPressShared/WordPressShared.xcodeproj/project.pbxproj

```diff
@@ -669,6 +677,7 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = A9D12BC8253E0DBA59BF461B /* Pods-WordPressShared.release-alpha.xcconfig */;
 			buildSettings = {
+				CLANG_ENABLE_MODULES = YES;
```

Comment:
Modules are enabled at the project level for WordPressShared. You can hit delete on the `Enable Modules (C and Objective-C)` entry in **Build Settings** for both targets (to unbold it). It's more of a nitpick and an irritation that Xcode lets you override a value with the same value 👎 .